docs/application/web/guides/device/system-information.md

```diff
@@ -234,6 +234,31 @@ To receive notifications on property value changes:
    var id = tizen.systeminfo.addPropertyValueArrayChangeListener('SIM', successCallback);
    ```
 
+
+<b>Remark:</b>
```

Comment:
We don't recommend use "Remark:" in guides, but we recommend "Note".
If the meaning is not quite different, please use "Note".

The following is the style of Note:
```
> **Note**
>
> This is Note style. Please write note like this.

Reply:
Done